### ISSUE TITLE
padding size and lightningmodule error

### ISSUE THREAD

**Issue by Starryskyswt** [rover-xingyu/Ha-NeRF#24]
![ERROR](https://user-images.githubusercontent.com/104331482/229514373-7ebdf4ab-1784-4fc7-aa0f-b9bccf193331.png)
I use the cache to run my code,but find two errors. I can not figure out what's the mistake,I doubt this is due to the cache produced by yor code:prepare_phototourism.py .... Besides,I use the brandenburg_gate dataset which I have successfully run on the nerf-w code. Could you please help me?

**Comment by rover-xingyu** [comment]
It looks weird. What downsampling scale (--img_downscale) did you use when you prepared and trained the dataset?

**Comment by Starryskyswt** [comment]
The instruction I entered is CUDA_VISIBLE_DEVICES=0,2,3  python3.6 train_mask_grid_sample.py  --root_dir /data0/dataset/nerf-w/brandenburg_gate  --dataset_name phototourism   --save_dir /data0/swt/Ha-NeRF   --img_downscale 2   --use_cache   --N_importance 64  --N_samples 64   --num_epochs 20 --batch_size 1024     --optimizer adam --lr 5e-4 --lr_scheduler cosine   --exp_name exp_HaNeRF_downscale2  --N_emb_xyz 15 --N_vocab 1500  --use_mask --maskrs_max 5e-2 --maskrs_min 6e-3 --maskrs_k 1e-3 --maskrd 0   --encode_a --N_a 48 --weightKL 1e-5 --encode_random --weightRecA 1e-3 --weightMS 1e-6  --num_gpus 3
Now I think it's due to the version of pytorch-lighnting,cause the graphics card of the server I am using is RTX3060. So torch==1.7.0 is not suitable,now I am using pip install torch==1.11.0+cu113 torchvision==0.12.0+cu113 torchaudio==0.11.0 --extra-index-url https://download.pytorch.org/whl/cu113 to run my code. Maybe the version of pytorch and pytorch-lighnting is not compatible? Could you please help me? 
![list](https://user-images.githubusercontent.com/104331482/229553377-e1e45084-bd4f-4023-ae4e-3130cd3f84ff.png)

**Comment by Starryskyswt** [comment]
Besides,I used downscale2 to prepare,so my dataset has a  cache folder and I think it's correct .
![cache](https://user-images.githubusercontent.com/104331482/229554093-f4ebb230-c13e-44c5-9b9c-e1219e4c1e52.png)

**Comment by BruceRussll** [comment]
Hi, I have the same problem. Have you solved it?
